cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js

```diff
@@ -76,6 +77,23 @@ var utils = {
   },
 
   /**
+   * getStateData - XHR state metadata
+   *
+   * @param {function} cb Function called with state data.
+   *
+   * @returns {function} Function called with state data.
+   */
+  getStateData: cb => {
```

Comment:
@marteki Rather than using a hardcoded list of default states, we're now pulling down a list from the API, identical to what we do with metros and counties. This will keep it fresh and fix the bug you described.

Reply:
Aw sweet, I've been half wanting to get rid of that hardcoded list for weeks... but, you know, priorities, and it wasn't really like the list of states in the USA was going to change. LITTLE DID WE KNOW.

I'll pull down and retest.